lib/vsc/utils/script_tools.py

```diff
@@ -281,6 +281,12 @@ def make_options(self, defaults=None):
 
         return ExtendedSimpleOption(options)
 
+    def ok(self, msg):
+        """
+        Convenience method that calls ExtendedSimpleOptions warning and exists with nagios OK exitcode
```

Comment:
`warning`?
anyway, exitcode values should come from first elelent of https://github.com/hpcugent/vsc-utils/blob/master/lib/vsc/utils/nagios.py#L66

Reply:
@stdweird fixed the typo